prime-router/src/main/kotlin/cli/ValidateTranslationSchemaCommand.kt

```diff
@@ -0,0 +1,73 @@
+package gov.cdc.prime.router.cli
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.enum
+import gov.cdc.prime.router.azure.BlobAccess
+import gov.cdc.prime.router.fhirengine.translation.TranslationSchemaManager
+import org.apache.logging.log4j.kotlin.Logging
+
+class ValidateTranslationSchemaCommand :
+    CliktCommand(
+    name = "validateSchemas",
+    help = "Commands to validate translation schemas"
```

Comment:
I could see it being useful for this help text to include more context about the pre-reqs for validation, especially mentioning sample input/output similarly to what is done in the `validateManagedSchemas` function comment.